src/collector.ts

```diff
@@ -88,22 +91,38 @@ class NaivePyParser {
 class ReqDependencyCollector implements IDependencyCollector {
     constructor(public classes: Array<string> = ["dependencies"]) {}
 
-    async collect(contents: string): Promise<Array<IDependency>> {
+    async collect(contents: string, manifestFile: string): Promise<Array<IDependency>> {
         let parser = new NaivePyParser(contents);
         return parser.parse();
     }
 
 }
 
 class NaiveGomodParser {
-    constructor(contents: string) {
-        this.dependencies = NaiveGomodParser.parseDependencies(contents);
+    constructor(contents: string, manifestFile: string) {
+        this.dependencies = NaiveGomodParser.parseDependencies(contents, manifestFile);
     }
 
     dependencies: Array<IDependency>;
 
-    static parseDependencies(contents:string): Array<IDependency> {
+    static parseDependencies(contents:string, manifestFile: string): Array<IDependency> {
         const gomod = contents.split("\n");
+
+        const vscodeRootpath = manifestFile.replace("file://", "").replace("/go.mod", "")
+        const gopackages = execSync(
+                            `cd "${vscodeRootpath}" && go list -f '{{ join .Imports "\\n" }}' ./...`,
+                            { maxBuffer: 1024 * 1200 }).toString().split("\n");
+        
+        let importPackages = gopackages.reduce((importPackages, importPckg) => {
+            if (importPckg.length > 0) {
+                if (importPackages.indexOf(importPckg) < 0)
+                    importPackages.push(importPckg)
+            }
+            return importPackages
+        }, []);
```

Comment:
Use Set to eliminate duplicate.
```suggestion
        }, new Set);
```